assets/js/components/KeyMetrics/ChangeMetricsLink.js

```diff
@@ -54,13 +62,59 @@ export default function ChangeMetricsLink() {
 
 	useChangeMetricsFeatureTourEffect( renderChangeMetricLink );
 
+	const intersectionEntry = useIntersection( trackingRef, {
+		threshold: 0.25,
+	} );
+	const [ hasBeenInView, setHasBeenInView ] = useState( false );
+	const inView = !! intersectionEntry?.intersectionRatio;
+
+	const { triggerSurvey } = useDispatch( CORE_USER );
+
+	const usingProxy = useSelect( ( select ) =>
+		select( CORE_SITE ).isUsingProxy()
+	);
+
+	const isKeyMetricsSetupCompletedBy = useSelect( ( select ) =>
+		select( CORE_SITE ).getKeyMetricsSetupCompletedBy()
+	);
+
+	const currentUserID = useSelect( ( select ) =>
+		select( CORE_USER ).getID()
+	);
+
+	useEffect( () => {
+		if ( inView && ! hasBeenInView ) {
+			if ( usingProxy ) {
+				triggerSurvey( 'view_kmw', {
+					ttl: WEEK_IN_SECONDS,
+				} );
+				if ( isKeyMetricsSetupCompletedBy === currentUserID ) {
+					triggerSurvey( 'view_kmw_setup_completed', {
+						ttl: WEEK_IN_SECONDS,
+					} );
+				}
+			}
+
+			setHasBeenInView( true );
+		}
+	}, [
+		inView,
+		viewContext,
+		hasBeenInView,
+		usingProxy,
+		triggerSurvey,
+		currentUserID,
+		isKeyMetricsSetupCompletedBy,
+	] );
+
 	if ( ! renderChangeMetricLink ) {
```

Comment:
The IB says
> Include `renderChangeMetricLink` in the condition, surveys shouldn't be triggered if the value is falsy

this wouldn't be necessary though if we can keep the former render-based triggering.